R/hydrofabric_io.R

```diff
@@ -75,7 +75,7 @@ read_hydrofabric = function(gpkg = NULL,
                             flowpaths = NULL,
                             realization  =  "all",
                             crs = NULL,
-                            verbose = TRUE){
+                            verbose = Sys.getenv("hydrofab_verbose") != "false"){
```

Comment:
This lets me kill verbose in tests

Reply:
This is in a lot of places. I can change them all if this is needed. Do we need to add the environment default?

Reply:
The form I used here makes it so it will only be FALSE if you have that environment variable set to "false" so it doesn't need to be defaulted. 

Adding it all over the place would be nice, yeah.